crud-s3/deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!bin/bash
+
+./bob/build.sh 3
```

Comment:
where is bob getting from?
it's not in the readme

Reply:
Line 23 of the Readme 👍 

Side note: I waffled back and forth between pulling the repo and adding a subtree of the bob repo to the functions-examples repo, since there will be more python examples made.